XenAdmin/Dialogs/VIFDialog.cs

```diff
@@ -126,80 +96,94 @@ private void LoadDetails()
                     promptTextBoxQoS.Text = ExistingVif.LimitString();
                     checkboxQoS.Checked = ExistingVif.qos_algorithm_type == VIF.RATE_LIMIT_QOS_VALUE;
                 }
-                flowLayoutPanelQoS.Enabled = checkboxQoS.Enabled = true;
 
-                panelLicenseRestriction.Visible = false;
+                tableLayoutPanel3.Enabled = checkboxQoS.Enabled = true;
+                labelInfo.Text = Messages.VIF_LICENSE_RESTRICTION;

```

Comment:
Unless I'm missing something, is this message ever visible?

Reply:
This was related to a license restriction that has been removed long ago (8ea7e73b55d3e75bd1c75cadfb6def6ddcd9add0), so we can drop it.